### DISCUSSION TITLE
bandwidth usage?

### DISCUSSION THREAD

**Discussion by shashankx86** [jlesage/docker-firefox#240] [General]
![image](https://github.com/jlesage/docker-firefox/assets/64682801/ca31a66f-b494-4703-9657-b3621a4a9d89)

approx bandwidth usage

**Comment by jlesage** [comment]
Sorry, what is the question ?

**Reply by shashankx86** [reply]
> Sorry, what is the question ?

Ok
I have limited bandwidth 
and I'm asking how much bandwidth it would use, like any guesses?

like
as far as i know vnc uses few mb per hr (~20 my own observation)

**Reply by jlesage** [reply]
Sorry, I don't have numbers. The bandwidth varies according the amount activity occurring on the screen.
I you want to minimize the data used, I would reduce the quality and increase the compression.
